lib/cmd/fh3/build.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   'desc' : i18n._('Builds a client application'),
   'examples' :
     [{
-      cmd : 'fhc build --project=<project> --app=<app-id> --cloud_app=<cloud-app-id> --tag=<tag> --environment=<environment> --destination=<destination> --version=<version> --config=<release|distribution> --privateKeyPass=<private-key-password> --certpass=<certificate-password> --download=<true|false> --bundleId=<credential bundle id> --cordova_version=<cordova version>  --git-tag=<tag name> --git-branch=<branch name> --git-commit=<commit hash>',
+      cmd : '[DEPRECATED] - fhc build --project=<project> --app=<app-id> --cloud_app=<cloud-app-id> --tag=<tag> --environment=<environment> --destination=<destination> --version=<version> --config=<release|distribution> --privateKeyPass=<private-key-password> --certpass=<certificate-password> --download=<true|false> --bundleId=<credential bundle id> --cordova_version=<cordova version>  --git-tag=<tag name> --git-branch=<branch name> --git-commit=<commit hash>',
```

Comment:
this will only show if the user enters the wrong command or use `fhc build help`, right? Can we print out a warning message in the implementation to ALWAYS show the deprecation notice and also link to the public [announcement](https://redhatmobilestatus.com/2018/09/26/immediate-deprecation-of-the-rhmap-client-binary-build-service-build-farm/)?